torch/distributed/rpc/api.py

```diff
@@ -592,9 +594,15 @@ def remote(to, func, args=None, kwargs=None, timeout=UNSET_RPC_TIMEOUT):
                 **kwargs,
             )
         else:
-            (pickled_python_udf, tensors) = _default_pickler.serialize(
-                PythonUDF(func, args, kwargs)
-            )
+            arg_cache_key = (func, args, str(kwargs))
```

Comment:
In the case where the cache is never used, can we test if there is any regression that might be caused by building this cache key (string conversion of a dict might be expensive)?